### PR TITLE
Add Google Cloud Functions label documentation

### PR DESCRIPTION
**Pull request by jbutz**
## What did you implement:

Related to serverless/serverless-google-cloudfunctions#94

Depends on serverless/serverless-google-cloudfunctions#95

Adding documentation for a PR I created related to Google Cloud Functions

## How did you implement it:

N/A

## How can we verify it:

N/A

## Todos:

- [X] Write tests
- [X] Write documentation
- [X] Fix linting errors
- [X] Make sure code coverage hasn't dropped
- [X] Provide verification config / commands / resources
- [X] Enable "Allow edits from maintainers" for this PR
- [X] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

**Comment by horike37** [comment]
Thanks @jbutz :+1: 
We can merge this PR after serverless/serverless-google-cloudfunctions#95 will be merged.

File: docs/providers/google/guide/functions.md [hunk 1]
```diff
@@ -96,3 +96,43 @@ exports.event = (event, callback) => {
...
+
+Labels can be applied globally, to all functions in your configuration file, and to specific functions.
+
+```yml
+# serverless.yml
+
+provider:
+  name: google
+  labels:
+    application: Severless Example
```
Comment (brendo) [inline docs/providers/google/guide/functions.md]: Typo here, `Severless` => `Serverless`

File: docs/providers/google/guide/functions.md [hunk 2]
```diff
@@ -96,3 +96,43 @@ exports.event = (event, callback) => {
...
+  second:
+    handler: httpSecond
+    events:
+      - http: path
+    labels:
+      application: Serverless Example - Documentation
+```
+
+With the above configuration the `httpFirst` function would have two labels applied, `application` and `team`.
+The value of the `application` label would be `Severless Example`, the value of the `team` label would be `GCF Team`.
```
Comment (brendo) [inline docs/providers/google/guide/functions.md]: And here :)

Comment (jbutz) [reply docs/providers/google/guide/functions.md]: @brendo Good catch on both of those, I'll get a fix in

**Comment by jbutz** [comment]
@horike37 Are you able to reach out to anyone to get serverless/serverless-google-cloudfunctions#95 reviewed?

**Comment by pmuens** [comment]
🤔 for some reason Travis got stuck. I tried to re-run the build without success...

@jbutz could you please merge master into this branch again or create another commit so that Travis kicks in again? Thanks!

**Comment by jbutz** [comment]
@pmuens 👍 On it

**Comment by jbutz** [comment]
@pmuens It looks like one of the tests timed out. Are you able to rerun that job or will I need a new commit to trigger it?

**Comment by horike37** [comment]
@jbutz 
Looks like the tests were just flaky, as I reran them and they have passed :smile:

**Comment by pmuens** [comment]
Awesome! Thanks @jbutz and @horike37 🙌

**Pull request merged**